### PR TITLE
LPS-160440 [Raylife AP] fix status chart request api

### PR DESCRIPTION
**Pull request by silasbsi**
Sub-task: [LPS-160440](https://issues.liferay.com/browse/LPS-160440)

### CODE REVIEW

**Comment by silasbsi** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @silasbsi

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-status/index.js [hunk 1]
```diff
@@ -29,93 +29,91 @@ export default function () {
...
+			CONSTANTS.STATUS.REVIEWED
+		);
+
+		const rejectedStatus = await getApplicationsStatus(
+			CONSTANTS.STATUS.REJECTED
+		);
+
+		const boundStatus = await getApplicationsStatus(CONSTANTS.STATUS.BOUND);
+
+		const cols = [
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-status/index.js:67]: What does "cols" mean? Maybe you should try a more clear name.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! Just added one suggestion :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-status/index.js (hunk 1 above)
Comment (silasbsi) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-status/index.js:67]: I do not know. :stuck_out_tongue:

**Review by joanagrego — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**